### ISSUE TITLE
Syntax highlighting issue with SET *

### ISSUE THREAD

**Issue by kurtlebreton** [jewuma/mumps-debug#20]
I've noticed the `SET *var=` syntax isn't being recognized. Please refer to the following code snippet:

```m
	new
	set *a=$$List()
	zwr
	quit
List()
	new list
	set list(1)="one"
	set list(2)="two"
	set list(3)="three"
	quit *list
``` 

The line `quit *list` doesn't produce syntax issues, however the `set *a=$$List()` is unrecognized.

**Comment by jewuma** [comment]
Problem is fixed at version 1.1.8
Thank you for reporting.

**Comment by kurtlebreton** [comment]
Many thanks, looks good :-)
